src/fides/ctl/core/deploy.py

```diff
@@ -65,10 +65,19 @@ def compare_semvers(version_a: List[int], version_b: List[int]) -> bool:
 
 def check_docker_version() -> bool:
     """Verify the Docker version."""
+    try:
+        run("docker info", stdout=PIPE, stderr=PIPE, check=True, shell=True)
+    except CalledProcessError:
+        raise SystemExit(
+            "Error: Could not run 'docker info'. Please ensure that Docker is running and try again."
+        )
+
     try:
         raw = run("docker --version", stdout=PIPE, check=True, shell=True)
     except CalledProcessError:
-        raise SystemExit("Error: Command 'docker' is not available.")
+        raise SystemExit(
+            "Error: Could not run 'docker --version'. Please ensure that Docker is running and try again."
```

Comment:
If we're catching the "Docker isn't running" case above, then a `docker --version` failure here would imply a different problem. This codepath would likely be accessed rarely, but on the off chance that it _is_ accessed, this error message would cause confusion. I have no idea how to make `docker --version` error while Docker is running, so I can't suggest better copy.

Reply:
Grouping this into the same try/except works better, tbh